src/client/game.cpp

```diff
@@ -882,7 +882,7 @@ class Game {
 	SoundMaker *soundmaker = nullptr;
 
 	ChatBackend *chat_backend = nullptr;
-	LogOutputBuffer m_chat_log_buf;
+	CaptureLogOutput m_chat_log_buf;
```

Comment:
shouldn't be this called CapturedLogOutput ?

Reply:
The CaptureLogOutput class actively captures logs by inserting itself into the logger's output list. It's not just storage for logs.